src/extensions/default/CodeFolding/foldhelpers/latex-fold.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Fold latex code regions
```

Comment:
@peterflynn I agree - and I've already tested moving the latex-fold function into the latex extension and that is works as expected. They can be safely removed from the main.js file (if you haven't sealed changes to 1.3 yet).
I also think the custom region should be disabled by default (and the menu option hidden).